scripts/generate.config.js

```diff
@@ -23,4 +23,9 @@ function generateConfig () {
   )
 }
 
-generateConfig()
```

Comment:
Isn't there too many try catch in the code, which all handle the errors the same way.
Can't they be all handled at a higher level ?

Reply:
Yep thanks I agree I will submit a commit